km52/Kholin_Nikita/laboratory1.sql

```diff
@@ -1,2 +1,114 @@
 -- LABORATORY WORK 1
 -- BY Kholin_Nikita
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+видаляти дані з таблиці
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+create user kholin identified by kholin_password;
+grant connect to kholin;
+grant delete on kholin.people to kholin;
+
```

Comment:
add tablespaces, quotas, first line witout ";" and connect(in roles) should be in ("")
--user
create user kholin identified by kholin_password
default tablespace "users"
temporary tablespace "temp";
--quotas
alter user artemenko quota 100M on users;
--roles
grant "connect" to kholin;